interp/constrintern.ml

```diff
@@ -359,13 +359,14 @@ let reset_hidden_inductive_implicit_test env =
          | (Inductive (params,_),b,c,d) -> (Inductive (params,false),b,c,d)
          | x -> x) env.impls }
 
-let check_hidden_implicit_parameters id impls =
+let check_hidden_implicit_parameters loc id impls =
```

Comment:
@ejgallego: Done, but quick `grep "^let" */*.ml{,4} */*/*.ml{,4} | grep "?loc " | wc` and `grep "^let" */*.ml{,4} */*/*.ml{,4} | grep " loc " | wc` gives respectively 69 and 197 occurrences, so this is far from being globally consistent yet.